include/phasar/PhasarLLVM/Utils/DefaultAnalysisPrinter.h

```diff
@@ -40,6 +40,7 @@ class DefaultAnalysisPrinter : public AnalysisPrinterBase<AnalysisDomainTy> {
 
 private:
   DataflowAnalysisResults<AnalysisDomainTy> AnalysisResults{};
+  llvm::raw_ostream &OS = llvm::outs();
```

Comment:
Use MaybeUniquePtr similar to the OnTheFlyAnalysisPrinter

Reply:
Done.